src/Restore.php

```diff
@@ -392,21 +392,6 @@ public function restoreBuckets(bool $checkBackend = true): void
                 $this->logger->warning(sprintf('Skipping bucket %s - %s', $bucketInfo->getId(), $e->getMessage()));
                 continue;
             }
-
-            if (count($bucketInfo->getMetadata())) {
```

Comment:
nechal jsem ty metadata v restorBucket() - bude to lepší protože metadata se obnoví až ve chvíli kdy se bucket skutečně založí